src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -79,6 +79,13 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
     case Event(id: WorkflowId, data) =>
       log.info(s"$tag: workflow ID UUID($id)")
       workflowManager ! SubscribeToWorkflow(id)
+      Runtime.getRuntime.addShutdownHook(new Thread() {
+        override def run(): Unit = {
+          workflowManager ! WorkflowAbort(id)
+          while(!isDone)
+            Thread.sleep(1000)
```

Comment:
We need this b/c the call to WorkflowAbort doesn't synchronously abort the jobs. If we leave this out then the JVM exits before the actors have time run the abort code. I know - I tried :(